apache/superset
# feat(pdf-support)!: Added download Dashboard and Chart as PDF support

### SUMMARY
Superset Supports downloading Dashboard, and Chart as an Image. It does not have an option to download the Dashboard or Chart as a PDF.
I have added this support using the [dom-to-pdf](https://www.npmjs.com/package/dom-to-pdf) npm package.

### BEFORE
<img width="1691" alt="image" src="https://github.com/apache/superset/assets/109591061/255943b8-d706-4bdf-bd1f-80e7379a7bc0">

### AFTER 
| Dashboard | Dashboard Chart | Explore |
| --- | --- | --- |
| ![Dashboard](https://github.com/apache/superset/assets/109591061/68ecb747-560c-4d7d-8bb3-4a5ed72e43b7) | ![Chart](https://github.com/apache/superset/assets/109591061/e93108c3-925b-4746-9a8a-e65625a8227c) | ![Explore](https://github.com/apache/superset/assets/109591061/dfafdfc9-3081-43f3-99d9-5e737b4ed772) |






Closes [#25446](https://github.com/apache/superset/issues/25446) 

### TESTING INSTRUCTIONS
- Create a dashboard and check if you have the option to download the Dashboard and Chart as a pdf

### ADDITIONAL INFORMATION
